.jazzy.yaml

```diff
@@ -1,13 +1,15 @@
-objc: true
+# Ordered by output of `jazzy --help config`
+
+output: docs/docs
 clean: true
-author: Stripe
-author_url: https://stripe.com
-github_url: https://github.com/stripe/stripe-ios
-github_file_prefix: https://github.com/stripe/stripe-ios/tree/master
```

Comment:
Could you leave this in, in case someone tries to run jazzy w/out using `ci_scripts/build_documentation.sh`?

I don't know if it'd work, I'm curious.